docker-compose/init-postgres-database.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 --username postgres --dbname postgres <<-EOSQL
```

Comment:
What about cassandra key spaces init?

Reply:
Maybe also we could imagine configuring the db prefix using en env variable, so that we can in some case start the app using alternative tables?